CellFabric/cell_fabric/pex.py

```diff
@@ -28,7 +28,7 @@ def run(self):
 
         # Create OrderedDict with NodeName -> layer, rect mappings
         for (layer, vv) in self.canvas.rd.store_scan_lines.items():
-            if layer not in self.canvas.pdk:
+            if layer.startswith('V' or 'M') == False or layer not in self.canvas.pdk:
```

Comment:
`layer.startswith('V' or 'M')` doesn't look right.

Change to:
`if not (layer.startswith('V') or layer.startswith('M')) or layer not in self.canvas.pdk:`